gcsmigrate/enqueue.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+from google.cloud import pubsub_v1
+from tqdm import tqdm
+from shared import make_generator_from_args, future_waiter
+
+TOPIC_ID = os.environ["TOPIC_ID"]
+GCP_PROJECT = os.environ["GCP_PROJECT"]
+
+VERIFY = b"verify"
+MIGRATE = b"migrate"
+
+# Trying to maximize throughput; we don't care about latency.
+batch_settings = pubsub_v1.types.BatchSettings(
+    max_messages=1000, max_bytes=10 * 1000 * 1000, max_latency=10
+)
+
+CLIENT = pubsub_v1.PublisherClient(batch_settings=batch_settings)
+
+TOPIC_PATH = CLIENT.topic_path(GCP_PROJECT, TOPIC_ID)
+
+
+def send_message(mogile_file, action):
+    """Send the mogile file as an pubsub message."""
+    return CLIENT.publish(TOPIC_PATH, mogile_file.to_json(), action=action)
+
+
+def queue_migrate(mogile_file):
+    """Queue mogile files for migration in pubsub."""
+    return send_message(mogile_file, MIGRATE)
+
+
+def queue_verify(mogile_file):
+    """Queue mogile files for verification in pubsub."""
+    return send_message(mogile_file, VERIFY)
+
+
+def main():
+    """Enqueue mogile file jobs to SQS for processing in AWS lambda."""
+    generator = tqdm(make_generator_from_args(sys.argv[2:]))
```

Comment:
I found it difficult to determine the expected arguments of this command since I needed to follow the logic down into `make_generator_from_args` (which isn't totally straightforward itself). For the sake of clarity, I think it would probably be better to utilize argparse here. But since this migration code is temporary, maybe a more appropriate level of effort is to just document the arguments in the docstring here.

Reply:
Yeah, it kind of sucks, but its just for one time use so I'm not sure how much its worth cleaning up. I added info about the args to the `main` docstring.

Reply:
84882eb